tool/tctl/common/status_command.go

```diff
@@ -160,5 +163,13 @@ func (c *StatusCommand) Status(ctx context.Context, client auth.ClientI) error {
 		}
 		fmt.Print(view())
 	}
+
+	if err := common.ShowClusterAlerts(ctx, client, os.Stderr, map[string]string{
+		types.AlertOnLogin: "yes",
```

Comment:
The original intent of the `OnLogin` label was specifically to control which alerts are shown once on login, sort of like an MOTD.  I don't think using that label constraint is very useful here.  IMO for a `status` command it makes more sense to either query by severity (e.g. everything MEDIUM or higher), or to just query all alerts, but only show maybe the "top N" alerts (i.e. sort the alerts, then trim the tail if there are more than N).